modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -146,16 +145,16 @@
 	alt_titles = list("Security Officer", "Security Operative", "Peacekeeper")
 
 /datum/job/security_sergeant
-	alt_titles = list("Security Sergeant", "Security Squad Leader", "Security Task Force Leader", "Security Fireteam Leader")
+	alt_titles = list("Security Sergeant", "Security Squad Leader", "Security Task Force Leader", "Security Fireteam Leader", "Security Enforcer") //why are these all so edgy, you're a mall cop
```

Comment:
Enforcer really doesnt sound like 'this is the **sergeant**', it just sounds like a milsim name for _generic_ officers...

Reply:
it was there before